tools/workspace-plugin/src/generators/migrate-converged-pkg/index.spec.ts

```diff
@@ -1644,3 +1644,13 @@ function getNormalizedPkgName(options: { pkgName: string; workspaceConfig: Works
 function getFixture(src: string) {
   return fs.readFileSync(path.join(__dirname, '__fixtures__', src), 'utf-8');
 }
+
+function assertAndReadNxJson(tree: Tree) {
+  const nxJson = readNxJson(tree);
```

Comment:
this returns object or null, thus I created this simple wrapper to get rid of annoying type assertions